src/layouts/layout.js

```
@@ -416,6 +416,94 @@ const layouts = {
+          0 0 45px #ff319b;
+        }
+      }
+
+      @media only screen and (max-width: 600px) {
+        .quote-container {
+          width: 100%;
+        }
+      }
+      `;
+      ;
+    },
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)